app/src/main/java/com/eveningoutpost/dexdrip/ShareModels/ShareRest.java

```diff
@@ -15,85 +30,64 @@
 import javax.net.ssl.TrustManager;
 import javax.net.ssl.X509TrustManager;
 
-import retrofit.RequestInterceptor;
-import retrofit.RestAdapter;
-import retrofit.android.AndroidLog;
-import retrofit.client.OkClient;
-import retrofit.converter.GsonConverter;
+import okio.Buffer;
+import retrofit.Callback;
+import retrofit.GsonConverterFactory;
+import retrofit.Retrofit;
 
 /**
  * Created by stephenblack on 12/26/14.
  */
 public class ShareRest {
-    public final static String TAG = ShareRest.class.getSimpleName();
+    public static String TAG = ShareRest.class.getSimpleName();
 
-    public static Gson gson = new GsonBuilder()
-            .excludeFieldsWithoutExposeAnnotation()
-            .create();
+    private String sessionId;
 
-    public static DexcomShareInterface jsonBodyInterface() {
-        RestAdapter adapter = authoirizeAdapterBuilder().build();
-        DexcomShareInterface dexcomShareInterface =
-                adapter.create(DexcomShareInterface.class);
-        return dexcomShareInterface;
-    }
+    private String username;
+    private String password;
+    private String serialNumber;
+    private DexcomShare dexcomShareApi;
 
-    public static DexcomShareInterface emptyBodyInterface() {
-        RestAdapter adapter = getBgAdapterBuilder().build();
-        DexcomShareInterface checkSessionActive =
-                adapter.create(DexcomShareInterface.class);
-        return checkSessionActive;
-    }
+    SharedPreferences.OnSharedPreferenceChangeListener preferenceChangeListener = new SharedPreferences.OnSharedPreferenceChangeListener() {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/matthias-granberry/xDrip-Experimental/file/1296698395/issues/source?bid=2735053&fileBranchId=2947762#l51)